chart/openfaas/README.md

```diff
@@ -288,6 +285,7 @@ Additional OpenFaaS options in `values.yaml`.
 | `exposeServices` | Expose `NodePorts/LoadBalancer`  | `true` |
 | `serviceType` | Type of external service to use `NodePort/LoadBalancer` | `NodePort` |
 | `basic_auth` | Enable basic authentication on the Gateway | `true` |
+| `basicAuthAutogen` | Generate basic authentication credentials | `false` |
```

Comment:
`generateBasicAuth`?

Reply:
Done

Reply:
Can you use `generateBasicAuth` so that it matches the way we put `exposeServices`